po/zh_CN.po

```diff
@@ -105,7 +105,7 @@ msgstr "共享内存"
 #: src/models/devices.js:65
 #, fuzzy
```

Comment:
hey @EricZhang456 , can you remove the "#, fuzzy" comment? It's no longer needed since you translated it now.